app/assets/src/components/views/amr_heatmap/AMRHeatmapView.jsx

```diff
@@ -295,6 +316,7 @@ export default class AMRHeatmapView extends React.Component {
         </p>
       );
     }
+    logAnalyticsEvent("AMRHeatmapView_heatmap_rendered");
```

Comment:
normally we don't log passive events and page views... if the same event is captured by clicking an already instrumented submit button or by loading the page then it is not needed.